int32 from_drone_id
int32 ct
float64 time
geometry_msgs/Transform transform_w_c
int32[] points
float32[] values
float32[] weights
